lighthouse-core/gather/gather-runner.js

```diff
@@ -204,8 +204,8 @@ class GatherRunner {
    * main document request failure, a security issue, etc.
    * @param {LH.Gatherer.PassContext} passContext
    * @param {LH.Gatherer.LoadData} loadData
-   * @param {LighthouseError|undefined} navigationError
```

Comment:
oops :)

Reply:
> oops :)

this one is so strange (for those curious, this currently resolves to the [`LighthouseError` typedef in popup.js](https://github.com/GoogleChrome/lighthouse/blob/1b8f4d845571a1266b14332dd577ad9e58cc9e91/clients/extension/scripts/popup.js#L9), which luckily is the right thing). Most typedefs don't end up in the global type space and will cause an error if you don't explicitly import them, so what chain of things makes it end up there?

Maybe because there's no `module.exports` (or any other kind of export) in `popup.js` so tsc doesn't treat it as a module?